### PR TITLE
Added menu item for copying link of an image

### LINKED ISSUE #20: Add menu item for copying link of an image

### PR DESCRIPTION
**Pull request by ruimonteiro93**
Fixes #20 

Added one more item to the context-menu, that let's the user get the link (url src) of an image. It is only displayed when the user clicks on an image - its visibility is initially set to false.

### CODE REVIEW

**Comment by sindresorhus** [comment]
See my comments in https://github.com/sindresorhus/electron-context-menu/pull/21.

**Comment by ruimonteiro93** [comment]
Got it. I just have a little question regarding it doing the same as the Copy Link Item. From what I understood, the copy link item writes a bookmark to the clipboard if the platform is Darwin OS, with its title being the Link text (props.linkText), and its url being the Link url (props.linkURL). If the platform is not Darwin, it copies the Link url to the clipboard, which mine already does for the image src. 
It so happens that , when clicking on an image, props.linkText comes empty (naturally), and I don't really know what to set as the Bookmark text for Darwin. Any idea?

**Comment by sindresorhus** [comment]
> when clicking on an image, props.linkText comes empty (naturally), and I don't really know what to set as the Bookmark text for Darwin. Any idea?

Just the URL. The point is that when it's pasted, it's clickable.

**Comment by ruimonteiro93** [comment]
Set both of the text and url of the bookmark as the image source URL, should be clickable when it's pasted now.

Regarding the documenting part, is there any place to update other than the readme?

File: index.js [hunk 1]
```diff
@@ -49,6 +49,16 @@ function create(win, opts) {
 					download(win, props.srcURL);
 				}
 			}, {
+				id: 'copyImageLink',
+				label: 'Copy Image Link',
```
Comment (sindresorhus) [inline index.js]: Should be named `Copy Image Address` to match browsers. Same with all other name references.

**Comment by sindresorhus** [comment]
It should be opt-in and be documented. See #21.

**Comment by sindresorhus** [comment]
ping

**Pull request closed without merging**